wdc-calculator-core/pom.xml

```diff
@@ -0,0 +1,69 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>hu.icellmobilsoft.wdc</groupId>
+		<artifactId>wdc</artifactId>
+		<version>1.1.0-SNAPSHOT</version>
+	</parent>
+
+	<groupId>hu.icellmobilsoft.wdc.calculator.core</groupId>
```

Comment:
`<groupId>hu.icellmobilsoft.wdc</groupId>`
es igy mar nem kell megadni mivel orokli. Tobbi pom is hasonlokeppen